base/system/diskpart/diskpart_msg.mc

```diff
@@ -6699,22 +6716,23 @@ Example:
     SELECT VOLUME=C:\\MountH
 .
 Language=Romanian
-    Selects the specified volume and shifts the focus to it.
+    Selectează volumul specificat și mută focalizarea pe el.
 
-Syntax:  SELECT VOLUME={<N> | <D>}
+Sintaxă:  SELECT VOLUME={<N> | <D>}
 
-    VOLUME=<N>  The number of the volume to receive the focus.
+    VOLUME=<N>  Numărul volumului care urmează să primească marcaj.
 
-    VOLUME=<D>  The drive letter or mounted folder path of the volume
-                to receive the focus.
+    VOLUME=<D>  Litera discului sau calea dosarului montat al volumului
+                ce urmează să primească marcaj.
 
-    If no volume is specified, the select command lists the current volume with
-    focus. You can specify the volume by number, drive letter, or mounted folder
-    path. On a basic disk, selecting a volume also gives the corresponding
-    partition focus. You can view the numbers of all volumes on the computer by
-    using the list volume command.
+    Dacă niciun volum nu e specificat, comanda SELECT listează volumul actual
+    cu marcaj. Puteți specififca volumul după număr, după litera discului sau
+    după calea dosarului montat. Pe un disc de bază, selectarea unui volum
+    oferă, de asemenea, focalizarea corespunzătoare pe partiție. Puteți
```

Comment:
In other instances, I noticed you've chose to translate "focus" (with adaptation) as "marcaj", so please take care of consistency.